content/docs/guides/first_steps.md

```diff
@@ -0,0 +1,178 @@
+
+If we were only interested in requests that resulted in HTTP code `200`, we could use this query to retrieve that information:
+
+```
+http_requests_total{code="200"}
+```
+
+To count the number of returned time series, you could write:
+
+```
+count(http_requests_total)
+```
+
+For more about the expression language, see the
+[expression language documentation](/docs/querying/basics/).
+
+## Using the graphing interface
+
+To graph expressions, navigate to `http://localhost:9090/graph` and use the "Graph" tab.
+
+For example, enter the following expression to graph the per-second HTTP request rate happening in the self-scraped Prometheus:
+
+```
+rate(http_requests_total[1m])
+```
+
+You can experiment with the graph range parameters and other settings.
+
+## Installing the Node Exporter
+
+Collecting metrics from Prometheus alone is not a good representation of Prometheus' capabilities. So let's use the Node Exporter to monitor our first resource. We're going to monitor the local Linux host that the Prometheus server is running on but you could monitor any Linux or OS X host. There's also [a WMI exporter](https://github.com/martinlindhe/wmi_exporter) for Microsoft Windows hosts too.
+
+[Download the latest release of the Node Exporter](/download/#node_exporter) of Prometheus for your platform, then extract it:
+
+```language-bash
+tar xvfz node_exporter-*.tar.gz
+cd node_exporter-*
+```
+
+The Node Exporter is a single binary, `node_exporter`, and has a configurable set of collectors for gathering various types of host-based metrics. By default, collectors gather [CPU, memory, disk, and other metrics](https://github.com/prometheus/node_exporter#enabled-by-default) and expose them for scraping.
+
+Let's start the node exporter now on our Linux host.
+
+```language-bash
+./node_exporter
+```
+
+The Node Exporter's metrics are available on port `9100` on the host at the `/metrics` path. In our case this is:
+
+`http://localhost:9100/metrics`
+
+You can browse to this URL to see the metrics being exposed.
+
+We now need to tell Prometheus about our new target.
+
+## Configuring Prometheus to monitor the host
+
+We will configure Prometheus to scrape this new target. To achieve this, add the following job definition to the `scrape_configs` section in our `prometheus.yml`:
+
+```
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['localhost:9090']
+  - job_name:  'node'
+    static_configs:
+      - targets: ['localhost:9100']
+        labels:
+          role: 'monitoring'
+```
+
+Our new job is called `node`. It scrapes a static target, `localhost` on port `9100`. You would replace this name with the name or IP address of the host you're monitoring. We also add a label, `role` with a value of `monitoring`, to our time series.
```

Comment:
Maybe clarify that you add role: monitoring because the server is a monitor server; and that label names and called could be anything

Reply:
Good point.